src/pages/gd/Claim/ClaimButton/styled.ts

```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components'
+import { ButtonPrimary } from "../../../../components/ButtonLegacy";
+
+export const StyledButton = styled(ButtonPrimary)`
```

Comment:
- all base/common styles for buttons etc should be defined by extendTheme https://docs.nativebase.io/customizing-components
- custom styles should be object (like in GD app) https://github.com/GeekyAnts/NativeBase/issues/4935#issuecomment-1144847331